src/components/organisms/TimingEventModal/TimingEventModal.tsx

```diff
@@ -246,26 +243,25 @@ export const TimingEventModal: React.FC<TimingEventModalProps> = ({
                 </div>
               </div>
 
-              <AdminSection title="Duration">
```

Comment:
Why remove `AdminSection` which is semantic and replace it with `<div  className="input-group">` which has a style from `global.scss`?

Reply:
It creates a H2 in the middle of form. That's not a convention in this form. I would rather update all the other divs than have this one being different. Given we're going to be redesigning all of this I'd rather not spend too much time on this kind of change.

Reply:
Not spending time === you don't change it in any way, even if there is h2 in the middle of the form. At least do not add `input-group`

Reply:
The h2 looks messy in the form with different font size etc.
![image](https://user-images.githubusercontent.com/88904/143858045-91b23505-5ac7-4e73-a11d-4e0a1954e657.png)
The rest of the form uses `input-group`.

What's the issue here? Is this about trying to remove bootstrap?

Reply:
It's about trying to remove `input-group`. If you can localize the few style props to `TimingEventModal__input-group` even by just copying it from the global, that would be good enough I guess.

Reply:
I can do that. Cheers :)